### PR TITLE
[G2M] Progress/indicator

### PR DESCRIPTION
**Pull request by hyx131**
**Changes:**
- add `ProgressIndicator` component

**Screenshots:**
Screen Shot 2022-03-25 at 9 40 50 AM

### CODE REVIEW

**Review by sallkall — APPROVED** [review]

**Comment by sallkall** [comment]
We will also need a vertical variant but good for now!

File: src/components/progress-indicator.js [hunk 1]
```diff
@@ -0,0 +1,108 @@
...
+import PropTypes from 'prop-types'
+import clsx from 'clsx'
+
+import { makeStyles } from '../utils/make-styles'
+import { CheckBold } from '../icons'
+
+
+const styles = makeStyles({
+  activeBorderColor: { borderColor: '#2242CD' },
+  inactiveBorderColor: { borderColor: '#9E9E9E' },
```
Comment (YizhiCatherineZhang) [inline src/components/progress-indicator.js:11]: @sallkall `#2242cd` and `#9E9E9E` These colours are not in our design colour system. Should we add it to the our colour guide?

**Review by YizhiCatherineZhang — APPROVED** [review]

File: src/components/progress-indicator.js (hunk 1 above)
Comment (sallkall) [reply src/components/progress-indicator.js:11]: Oh this is what we talked about during design sync this morning, @vlatawiec and I already fixed it and shared with selin.

Comment (YizhiCatherineZhang) [reply src/components/progress-indicator.js:11]: I heard but I didn't match the content with this PR. My bad, Tyty.

**Pull request merged**